test/FunctionalTests/Microsoft.OData.Core.Tests/Evaluation/ODataEntryMetadataContextTest.cs

```diff
@@ -209,14 +209,6 @@ public void ETagPropertiesShouldReturnEmptyForEntryMetadataContextWithoutModelAn
             this.entryMetadataContextWithoutModel.ETagProperties.Should().BeEmpty();
         }
 
-        [Fact(Skip = "we should remove this case because we need model to calc ETag")]

```

Comment:
I looked at rewriting the test, it looked like it was impacted by the 7.x API change. I took the hint from the skip comment and just removed them. We don't seem to have test gaps for ETags should removing should be safe.